unpick-definitions/mining_levels.unpick

```diff
@@ -0,0 +1,13 @@
+v2
+
+constant mining_levels net/fabricmc/yarn/constants/MiningLevels HAND
+constant mining_levels net/fabricmc/yarn/constants/MiningLevels WOOD
+constant mining_levels net/fabricmc/yarn/constants/MiningLevels STONE
+constant mining_levels net/fabricmc/yarn/constants/MiningLevels IRON
+constant mining_levels net/fabricmc/yarn/constants/MiningLevels DIAMOND
+constant mining_levels net/fabricmc/yarn/constants/MiningLevels NETHERITE
+
+target_method net/minecraft/item/ToolMaterials <init> (IIFFILjava/util/function/Supplier;)V
```

Comment:
The reason this is not working is there is no such constructor in the bytecode. Enum constructor parameters always start with `int ordinal, String name`.

Reply:
More exactly, the first 2 params are `String, int`, like https://github.com/FabricMC/yarn/blob/c44013c861fd5bd68ddfe9af2eeb6ed8a263c067/mappings/net/minecraft/entity/SpawnGroup.mapping#L19

Yeah, even in enigma notation, enum ctor param name slot start with 3, where slot 0 is `this`, slot 1 is field name, slot 2 is ordinal

Reply:
Ah okay, the MCDev IntelliJ Plugin's `Copy Mixin Target Reference` gives the signature I used, guess that doesn't factor in enum stuff properly.